src/server/service/kv.rs

```diff
@@ -1793,12 +1811,7 @@ fn future_flashback_to_version<
             // in `RegionLocalState` and region's meta, and when that
             // admin cmd is applied, will update the memory
             // state of the flashback
-            send_flashback_msg::<T, E>(
-                &raft_router,
-                req.get_context(),
-                AdminCmdType::FinishFlashback,
-            )
-            .await?;
+            send_flashback_msg::<T, E>(&raft_router, &ctx, AdminCmdType::FinishFlashback).await?;
```

Comment:
If tikv is down after the 2pc lock is released, but the completion flashback has not been sent yet.
At this point tidb will retry the flashback, but there is a window period when it will advance the CDC.